### DISCUSSION TITLE
Embedding as iframe

### DISCUSSION THREAD

**Discussion by JarydX** [chatwoot/chatwoot#5897] [Q&A] [answered]
Hi,

is there a way to integrate the agent webapp into other webpages (in this case our ticket system) via iframe?
At the moment i am getting an CORS-error for that...

Maybe there is a config file or something? We are using the selfhosted variant.

Thank you!

**Comment by sojan-official** [comment, marked as answer]
We don't have a configuration. but the CORs settings are controller by the following file.

ref: https://github.com/chatwoot/chatwoot/blob/develop/config/initializers/cors.rb
